Lib/test/test_csv.py

```diff
@@ -954,6 +954,7 @@ class TestSniffer(unittest.TestCase):
 'Tommy''s Place'+ Blue Island'+ 'IL'+ '12/28/02'+ 'Blue Sunday/White Crow'
 'Stonecutters ''Seafood'' and Chop House'+ 'Lemont'+ 'IL'+ '12/19/02'+ 'Week Back'
 """
+    sample10 = "Date;Value\r2010-01-01;10\r2010-01-01;20"
```

Comment:
Either move this line into the test function or put the literal into the `sniff` call.  See comment below.